### PR TITLE
Create README.rst for the Datadog trace exporter

### PR DESCRIPTION
**Pull request by nbutton23**
Created a README.rst for the Datadog trace Exporter based on the Azure Monitor exporter

### CODE REVIEW

File: contrib/opencensus-ext-datadog/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Changelog
 
-## Unreleased
-- Initial version.
+## 0.1.0
```
Comment (reyang) [inline contrib/opencensus-ext-datadog/CHANGELOG.md]: This one hasn't been released to PyPI (e.g. https://pypi.org/project/opencensus/) yet. We should leave it as `Unreleased`.

**Comment by nbutton23** [comment]
@googlebot I fixed it.

**Comment by nbutton23** [comment]
@reyang What is the process for releasing this?

**Comment by lzchen** [comment]
@nbutton23 
I can do a minor release for `opencensus-ext-datadog`. Are you waiting on the package to be released before uploading to PyPI or is it already on PyPi?

**Comment by nbutton23** [comment]
I am new to Python so don't know the release and packaging process.  So I have never push anything to PyPI

**Pull request merged**